packages/kokoas-client/src/pages/projRegisterV2/sections/LedgerInformation/LedgerRadioButton.tsx

```diff
@@ -35,8 +33,7 @@ export const LedgerRadioButton = ({
           <RadioGroup
             row
             {...otherValue}
-            defaultValue={defaultValue}
-            value={value || ''}
+            value={value || 'ANDPAD'} // 初期値を'ANDPAD'とする
```

Comment:
```suggestion
            value={value}
```

変換段階で、空のケースがなくなると思いますが、勘違いがあれば、教えてください。

Reply:
ご指摘ありがとうございます。
convertProjToForm.tsの修正により、私も空のケースはなくなる認識です。
[0e34cc4](https://github.com/Lorenzras/yumecoco-monorepo/pull/1014/commits/0e34cc4f5412131988f0226cde4cd7bc1595121a)にて修正しましたので、ご確認をお願いいたします。